ansible/playbooks/roles/preflight/tasks/apply.yml

```diff
@@ -32,6 +32,41 @@
     - groups.kubernetes_master | length > 0
     - "'applications' in roles_with_generated_vars" # there is a case with disabled applications by custom feature mapping
 
+- name: Kubernetes checks
```

Comment:
Just thinking if we can already move the preflight into the K8s role? As we do with Postgres I think?

Reply:
Was thinking about that, but we have a few separate k8s-related roles, so if there is such proposal, I would like to discuss where  to place that.

Reply:
I would move those checks to role but we have 3 K8s roles.
1) memory: seems to be common so can be moved to kubernetes_common
2) CPUs: don't know if this is common, based on docs it seems that yes but in K8s source code I found preflight checks applicable only to control-plane nodes. So we may move this to kubernetes_master role for now (safer option) or kubernetes_common (to be consistent with the docs).
3) swap: we disable it in common role, which is run after preflight, so I would not add any preflight here (too early) and later I don't see significant gain since it would be duplication of K8s preflight check. What we can consider is adding assertion task just after disabling swap (to verify).